### PR TITLE
Replace values that contain repository with hub

### PR DESCRIPTION
**Pull request by sdake**
The main Istio hub is listed as hub while others were
listed as repository.  Unify on terminology "hub".

### CODE REVIEW

File: install/kubernetes/helm/istio-remote/charts/security/templates/cleanup-old-ca.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ spec:
       serviceAccountName: istio-cleanup-old-ca-service-account
       containers:
         - name: hyperkube
-          image: "{{ .Values.global.hyperkube.repository }}:{{ .Values.global.hyperkube.tag }}"
+          image: "{{ .Values.global.hyperkube.hub }}:{{ .Values.global.hyperkube.tag }}"
```
Comment (costinm) [inline install/kubernetes/helm/istio-remote/charts/security/templates/cleanup-old-ca.yaml]: For consistency we still need the middle part - hub doesn't include image.

File: install/kubernetes/helm/istio-remote/values.yaml [hunk 2]
```diff
@@ -62,7 +62,7 @@ global:
 
   # Not recommended for user to configure this. Hyperkube image to use when creating custom resources
   hyperkube:
-    repository: quay.io/coreos/hyperkube
+    hub: quay.io/coreos/hyperkube
```
[the remark this answers is not in the document]
Comment (sdake) [reply install/kubernetes/helm/istio-remote/values.yaml]: This fell off my radar.  I will submit a new PR against 1.0 with the appropriate changes.

**Comment by gyliu513** [comment]
@sdake can we close this one, as it was already handled in https://github.com/istio/istio/pull/7187 for 1.0 and will sync up to master as well.

**Comment by sdake** [comment]
already merged.

**Pull request closed without merging**